components/filter-field/src/filter-field.ts

```diff
@@ -429,6 +448,11 @@ export class DtFilterField<T> implements AfterViewInit, OnDestroy, OnChanges {
     fromEvent(this._inputEl.nativeElement, 'input')
       .pipe(
         takeUntil(this._destroy),
+        map(() => this._inputEl.nativeElement.value),
```

Comment:
please put all the operators before the takeUntil.

The general rule is that `takeUntil` should be the last operator in the sequence. Only when there is a `shareReplay` operator it should be second last operator.

In this case, it would be not that bad, but please for consistency and to avoid later bugs.

There would be a lint rule for that:
https://github.com/cartant/rxjs-tslint-rules#rxjs-no-unsafe-takeuntil